specs/phase1/beacon-chain.md

```diff
@@ -557,26 +308,30 @@ def get_shard_committee(beacon_state: BeaconState, epoch: Epoch, shard: Shard) -
         indices=active_validator_indices,
         seed=seed,
         index=shard,
-        count=get_active_shard_count(beacon_state),
+        count=get_active_shard_count(beacon_state, epoch),
```

Comment:
Not directly related to changes in this PR, but there are a few functions where the parameter is called `beacon_state` and we usually use `state`. Maybe worth changing for consistency?